test/e2e/autoscaling/dns_autoscaling.go

```diff
@@ -104,7 +105,8 @@ var _ = SIGDescribe("DNS horizontal autoscaling", func() {
 
 	// This test is separated because it is slow and need to run serially.
 	// Will take around 5 minutes to run on a 4 nodes cluster.
-	f.It(f.WithSerial(), f.WithSlow(), "kube-dns-autoscaler should scale kube-dns pods when cluster size changed", func(ctx context.Context) {
+	// TODO(upodroid) This test will be removed in 1.33 when kubeup is removed
+	f.It(f.WithSerial(), f.WithSlow(), feature.KubeUp, "kube-dns-autoscaler should scale kube-dns pods when cluster size changed", func(ctx context.Context) {
```

Comment:
can we just tag all the problem tests that we want to tag in this PR?
https://github.com/kubernetes/kubernetes/issues/120989